src/functions/certGenInit.ts

```diff
@@ -30,41 +27,30 @@ const certGenInit: Handler = async (
 
   // Instantiate the Simple Queue Service
   const sqService: SQService = new SQService(new SQS());
-  const sendMessagePromises: Array<Promise<
-    PromiseResult<SendMessageResult, AWSError>
-  >> = [];
+  const sendMessagePromises: Array<Promise<PromiseResult<SendMessageResult, AWSError>>> = [];
 
   certGenFilteredRecords.forEach((record: any) => {
-    try {
       sendMessagePromises.push(
         sqService.sendCertGenMessage(JSON.stringify(record))
       );
-    } catch (error) {
-      console.log("failed on forEach for certGenFilteredRecords");
-      console.log(record);
-      throw error;
-    }
   });
 
   expandedRecords.forEach((record: any) => {
-    try {
       sendMessagePromises.push(
         sqService.sendUpdateStatusMessage(JSON.stringify(record))
       );
-    } catch (error) {
-      console.log("failed on forEach for expandedRecords");
-      console.log(record);
-      throw error;
-    }
+
   });
 
   return Promise.all(sendMessagePromises).catch((error: AWSError) => {
     console.error(error);
     console.log("expandedRecords");
-    console.log(expandedRecords);
+    console.log(JSON.stringify(expandedRecords));
```

Comment:
can be reduced to console.log("expandedRecords", JSON.stringify(expandedRecords));
same below

Reply:
I think this is to ensure, if for some reason stringify process fails to convert because of corrupted data, we still have a logline before that to pinpoint exact point of failure.